src/FSharp.Data.GraphQL.Shared/Errors.fs

```diff
@@ -216,7 +257,15 @@ type GQLProblemDetails = {
             | :? ICoerceGQLError as error -> error.VariableMessage + error.Message
             | _ -> error.Message
 
-        GQLProblemDetails.Create (message, path, extensions)
+        match error with
+        | :? IGQLExceptionError as exceptionError ->
+            match extensions with
+            | Include x ->
+                GQLProblemDetails.CreateOfException(exceptionError.Message, exceptionError.Exception, path, extensions = x)
+            | Skip ->
+                GQLProblemDetails.CreateOfException(exceptionError.Message, exceptionError.Exception, path)
```

Comment:
We can make it simpler. Anyway `option` is created under the hood
```suggestion
            let x = extensions |> Skippable.toOption
            GQLProblemDetails.CreateOfException(exceptionError.Message, exceptionError.Exception, path, ?extensions = x)
```

Reply:
This doesn't work. Believe me, I tried this 🙂 . It doesn't expect an `option`, although the parameter is `option` under the hood. It's apparently characteristic of the `?extensions` syntax.

Reply:
I've made it work in my code but I implemented a better change

Reply:
See PR to your branch